ginipaybank/src/main/res/layout/gpb_fragment_line_item_details.xml

```diff
@@ -1,166 +1,218 @@
+        app:layout_constraintTop_toTopOf="@id/gpb_checkbox"
         tools:text="1 item selected" />
 
     <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/gpb_description_container"
-        style="@style/Widget.MaterialComponents.TextInputLayout.FilledBox.Dense"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/gpb_checkbox"
-        android:layout_marginBottom="@dimen/gpb_digital_invoice_line_item_details_description_margin_bottom"
+        android:layout_height="64dp"
+        android:layout_marginTop="@dimen/gpb_digital_invoice_line_item_details_description_margin_top"
+        android:hint="@string/gpb_digital_invoice_line_item_details_description_label"
         android:textColorHint="@color/gpb_digital_invoice_line_item_details_input_field_hint_text"
+        app:boxBackgroundColor="@color/gpb_digital_invoice_box_background_color"
+        app:endIconDrawable="@drawable/gpb_digital_invoice_input_clear_icon"
+        app:endIconMode="clear_text"
+        app:hintTextAppearance="@style/GiniCaptureTheme.DigitalInvoice.LineItemDetails.InputField.Hint.TextStyle"
         app:hintTextColor="@color/gpb_digital_invoice_line_item_details_input_field_hint_text"
-        app:boxBackgroundColor="@android:color/transparent">
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/gpb_checkbox">
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/gpb_description"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:hint="@string/gpb_digital_invoice_line_item_details_description_label"
+            android:layout_height="match_parent"
             android:inputType="text"
             android:nextFocusDown="@id/gpb_quantity"
+            android:paddingStart="0dp"
+            android:paddingEnd="0dp"
+            android:paddingBottom="0dp"
             android:textAppearance="@style/GiniCaptureTheme.DigitalInvoice.LineItemDetails.InputField.TextStyle"
             tools:text="Nike Sportswear Air Max 97 - Sneaker Low" />
 
     </com.google.android.material.textfield.TextInputLayout>
 
-    <LinearLayout
-        android:id="@+id/gpb_quantity_and_gross_price_container"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/gpb_description_container"
-        android:layout_marginBottom="@dimen/gpb_digital_invoice_line_item_details_quantity_and_gross_price_margin_bottom"
-        android:orientation="horizontal">
 
-        <com.google.android.material.textfield.TextInputLayout
-            android:id="@+id/gpb_quantity_container"
-            style="@style/Widget.MaterialComponents.TextInputLayout.FilledBox.Dense"
+    <com.google.android.material.textfield.TextInputLayout
```

Comment:
This should have a style defined so the client can hook into it to customise input fields